### ISSUE TITLE
New Schematics

### ISSUE THREAD

**Issue by pandon101** [AstronIsTaken/du-craft#14]
Any chance you will add schematic and schematic cost?

**Comment by AstronIsTaken** [comment]
No, I don't think so, unless I come back playing again. But for that NQ should add reason to play to the game.
